### DISCUSSION TITLE
请教大神们，xray未运行咋解决，报错内容在下面，十分感谢

### DISCUSSION THREAD

**Discussion by linnana22** [XTLS/Xray-core#3747] [Q&A]
Xray 1.7.5 (Xray, Penetrates Everything.) Custom (go1.20 linux/amd64)

A unified platform for anti-censorship.

2024/08/31 20:30:03 [Info] infra/conf/serial: Reading config: bin/config.json

Failed to start: main: failed to load config files: [bin/config.json] > infra/conf: Failed to build TLS config. > infra/conf: failed to parse certificate > open /etc/x-ui/server.crt: no such file or directory

**Comment by linnana22** [comment]
已解决

**Reply by fxasdasdasd** [reply]
怎么解决的？

**Reply by LADYBUG0124** [reply]
重启VPS
